js-basic/Array/array.js

```diff
@@ -0,0 +1,55 @@
+let a = [18, 23, 23, 4, 4, 5];
+let k = ['Aditya', 'vinod', 'rakesh', 'crab'];
+
+let People = [
+  { name: 'aditya', age: 23, number: 9672362323, email: 'Adi@gmail.com' },
+  { name: 'Ravneet', age: 7, number: 2172362323, email: 'Rasv@gmail.com' },
+  { name: 'suitya', age: 39, number: 1232362323, email: 'su@gmail.com' },
+  { name: 'Nitya', age: 30, number: 8922362323, email: 'nitya@gmail.com' },
+];
+
+function ages(um) {
+  return um > 18;
+}
+
+// a.shift(); remove from start chnage origanal array "// " it will make time because of it will shift all value
+
+// a.push(1);  remove from last of array
+
+// a.unshift(2);  it will add on start chnage in orginal array
+
+// a.pop(); it will remove last element odf array
+
+// let b = a.concat(k); add two more than array in new array it will not change orignal array
+
+// a.splice(1, 2,4);callback values index deltecount addnum
+
+// let n = a.slice(0, 2);  it will not chnage and original array it will take out value from and array
+
```

Comment:
Slove these methods and add results and examples as well

Reply:
all the method add & result